.simplecov

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
+require 'securerandom'
+
+random = SecureRandom.uuid
+SimpleCov.command_name(random)
+SimpleCov.coverage_dir("coverage_#{random}")
```

Comment:
With this change, a new coverage directory will be created every time a test run is complete. This prevents results from being overwritten by subsequent runs. Inspo from [ENOSUCHBLOG](https://blog.yossarian.net/2018/04/01/Code-coverage-with-Simplecov-across-multiple-processes). 

The folder names, however, look a bit gnarly: `/coverage_9ea424f4-ce16-4c07-ae67-2d15d42f66621`

We haven't been doing anything with them though. Are we comfortable with using this approach to create unique names?

Reply:
That makes sense to me. We have so many different runs that this does seem like the most straightforward way to do that.